content/working-groups/prioritization/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Prioritization Working Group
+type: docs
+---
+
+# Prioritization Working Group
+![working group status: active][status]
+
+ Triaging bugs, mainly deciding if bugs are critical (potential release blockers) or not.
+
+- **Leads:**  [@spastorino][spastorino] and [@wesleywiser][wesleywiser]
+- **FAQ:** [FAQ]({{< relref "/working-groups/prioritization/FAQ" >}})
+
+[status]: https://img.shields.io/badge/status-active-brightgreen.svg?style=for-the-badge
+[spastorino]: https://github.com/spastorino
+[wesleywiser]: https://github.com/wesleywiser
+
+## What is the goal of this working group?
+
+This working group aims to accomplish the following:
+
+- Processing 'nominations' and routing bugs to folks who can fix them
+- Identifying *critical* bugs and monitoring them to ensure they are making progress
+- Identifying the agenda for compiler team triage meetings
+  - Critical issues that are not making progress
+  - Issues where bugs are nominated for needing wider discussion
+    - Ideally, crystallize
```

Comment:
incomplete sentence

Reply:
Yeah, hehe I had the same thought when I copied that from the Zulip conversation. Going to remove this.